extensions/wikia/WhereIsExtension/templates/list.tmpl.php

```diff
@@ -34,88 +34,99 @@
 			?>
 		</select>
 		<p class="likeValue"  style="display:none">
-			<?php print wfMsg('whereisextension-search-like-value') ?>
+			<?= wfMessage( 'whereisextension-search-like-value' )->escaped() ?>
 		</p>
 
-		<input value="<?php echo $formData['likeValue']; ?>" type="text" name="likeValue" class="likeValue"  style="display:none" />&nbsp;
-		<input type="submit" value="<?php print wfMsg('whereisextension-submit') ?>"/>
+		<input value="<?= Sanitizer::encodeAttribute( $formData['likeValue'] ) ?>" type="text" name="likeValue" class="likeValue"  style="display:none">&nbsp;
+		<input type="submit" value="<?= wfMessage( 'whereisextension-submit' )->escaped() ?>">
 	</form>
 
 	<br/>
-	<?php print wfMsg('whereisextension-filter') ?>
+	<?= wfMessage( 'whereisextension-filter' )->escaped() ?>
 	<br/>
 	<select id="groupSelect" name="group">
 		<option value="0" selected="selected">
-			<?php print wfMsg('whereisextension-all-groups') ?>
+			<?= wfMessage( 'whereisextension-all-groups' )->escaped() ?>
 		</option>
-		<? foreach ($formData['groups'] as $key => $value) {
-			$selected = $key == $formData['selectedGroup'] ? ' selected="selected"' : '';
+		<? foreach ( $formData['groups'] as $key => $value ) {
+			$selected = $key == $formData['selectedGroup'] ? ' selected' : '';
 		?>
-		<option value="<?php print $key ?>"<?php print $selected ?>><?php print $value ?></option>
+		<option value="<?= Sanitizer::encodeAttribute( $key ) ?>"<?= $selected ?>>
+			<?= htmlspecialchars( $value ) ?>
+		</option>
 		<? } ?>
 	</select>
 	<br/>
-	<label for="withString"><?php print wfMsg('whereisextension-name-contains') ?></label>
+	<label for="withString"><?= wfMessage( 'whereisextension-name-contains' )->escaped() ?></label>
 	<br/>
 	<input type="text" name="withString" id="withString" size="18" />
 
 	<?php
-	if (!empty($formData['wikis']) && count($formData['wikis'])) {
+	if ( !empty( $formData['wikis'] ) && count( $formData['wikis'] ) ) {
 		?>
-		<h3 id="headerWikis"><?php print wfMsg('whereisextension-list', $formData['count'] ) ?></h3>
-		<form method="post" action="<?php print $formData['actionURL'] ?>" name="wikiSelectForm">
+		<h3 id="headerWikis"><?= wfMessage( 'whereisextension-list', $formData['count'] )->escaped() ?></h3>
+		<form method="post" action="<?= Sanitizer::encodeAttribute( $formData['actionURL'] ) ?>" name="wikiSelectForm">
 			<ul>
-			<?php
-			$front = '&nbsp;<a href="' . Title::makeTitle( NS_SPECIAL, 'WikiFactory' )->getFullUrl() . '/';
-			$back = '/variables/' . $formData['vars'][ $formData['selectedVar'] ] . '">[edit]</a>';
-			foreach($formData['wikis'] as $wikiID => $wikiInfo) {
-				$editURL = $front . $wikiID . $back;
+				<?
+					foreach( $formData['wikis'] as $wikiID => $wikiInfo ) :
+						$style = !$wikiInfo['p'] ? 'style="color: red"' : '';
+						$editURL = Title::makeTitle( NS_SPECIAL, 'WikiFactory' )->getFullURL() . '/' . $wikiID;
+						$editURL .= '/variables/' . $formData['vars'][ $formData['selectedVar'] ]
```

Comment:
This can be simplified to: `SpecialPage::getTitleFor( 'WikiFactory', "{$wikiID}/variables/{$formData['vars'][ $formData['selectedVar'] ]} )->getFullURL();`